File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/packets/PacketPing.java
package com.gmail.marcosav2010.communicator.packet.packets;

import com.gmail.marcosav2010.communicator.packet.Packet;
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketDecoder;
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketEncoder;
import lombok.NoArgsConstructor;

@NoArgsConstructor
public class PacketPing extends Packet {

    @Override
    protected void encodeContent(PacketEncoder out) {
    }

    @Override
    protected void decodeContent(PacketDecoder in) {
    }
}

File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/packets/PacketPing.kt
package com.gmail.marcosav2010.communicator.packet.packets

import com.gmail.marcosav2010.communicator.packet.Packet
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketDecoder
import com.gmail.marcosav2010.communicator.packet.wrapper.PacketEncoder

class PacketPing : Packet() {

    override fun encodeContent(out: PacketEncoder) {}
    override fun decodeContent(`in`: PacketDecoder) {}
}